lib/rack/session/dalli.rb

```diff
@@ -55,7 +55,7 @@ def get_session(env, sid)
 
       def set_session(env, session_id, new_session, options)
         return false unless session_id
-        expiry = options[:expire_after]
+        expiry = new_session['expire_after'] || options[:expire_after]
```

Comment:
Why is the key for `new_session` a string, while the key for `options` is a symbol?